internal/streamingstore/streamingstore.go

```diff
@@ -0,0 +1,204 @@
+package streamingstore
```

Comment:
if server is migrating to `pkg`, feels like this should be `pkg/internal` rather than top level `internal`

Reply:
```
server/http/ipfs.go:11:2: use of internal package github.com/filecoin-project/lassie/pkg/internal/streamingstore not allowed (compile)
```

will have to wait till server is moved since only packages under pkg can use it once it's in there.